pages/helpers.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+export function readData<T extends object>(json_name: string): T[] {
+  const json = fs
+    .readFileSync(path.join(process.cwd(), `data/${json_name}.json`))
+    .toString();
+  return JSON.parse(json) as T[];
```

Comment:
Remind me to tell you about `io-ts` and runtime type decoding of JSON at some point (more for funzies, it'd be overkill on this project 😂).

Reply:
Ooh that sounds super cool. I will be sure to!